### ISSUE TITLE
Prototype Dom/CLC in Drupal

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#29104]
## Issue Description
We need to work with actual Drupal data to mockup Dom and CLC pages

---
## Tasks
- [ ] Work with Michelle to determine layout/components for Doms/CLCs

## Acceptance Criteria
- [ ] Sample prototype page for Dom
- [ ] Sample prototype page for CLC

**Comment by mmiddaugh** [comment]
blocked by issues with review instances and now no longer needed
